tool/coverage.sh

```diff
@@ -9,8 +9,11 @@ set -e
 
 # Gather coverage.
 pub global activate remove_from_coverage
-pub global activate dart_coveralls
+pub global activate coverage
 # Generate coverage report.
-pub global run dart_coveralls calc test/test_coverage.dart > lcov.info
+dart --no-sound-null-safety --pause-isolates-on-exit --disable-service-auth-codes --enable-vm-service=3000 test/test_coverage.dart &
```

Comment:
We could land this separately if you want.

Reply:
Since this is already done, I think we can let it be. If that's okay with you.